modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/hemophage.dm

```diff
@@ -472,6 +473,15 @@
 	body.vomit(lost_nutrition = 0, stun = FALSE, distance = 1, force = TRUE, purge_ratio = HEMOPHAGE_VOMIT_PURGE_RATIO)
 	return ..()
 
+/obj/item/organ/internal/stomach/hemophage/on_life(delta_time, times_fired)
+	var/datum/reagent/blood/blood = reagents.get_reagent(/datum/reagent/blood)
+	if(blood)
+		. = ...() // yes this is cursed, but we don't want to call /obj/item/organ/internal/stomach 's on_life here
```

Comment:
I never knew this worked, that's really funny.

Reply:
We don't want to call the parent stomach because otherwise organ failure and such won't happen if it does; @vinylspiders worked w/ me on this bit.